lib/cereal/postgres/sql_schema.go

```diff
@@ -119,6 +119,7 @@ CREATE TABLE cereal_tasks (
 
 CREATE TABLE cereal_task_results (
     id BIGSERIAL PRIMARY KEY,
+    task_id BIGINT NOT NULL,
```

Comment:
Should we also add a unique constraint?

Reply:
i think i like that. i'm guessing we wouldn't need an index since this table should be pretty small?